btr-web/btr-main-app/cypress/e2e/pages/add-individual/external-influence.cy.ts

```diff
@@ -1,42 +0,0 @@
-describe('pages -> Add individual', () => {
-  beforeEach(() => {
-    cy.visit('/')
-  })
-
-  it('renders the External Influence section and default radio is selected', () => {
```

Comment:
again just verifying that the design wants to remove this from the form? If not we should keep these tests